### ISSUE TITLE
Account keeps logging out.

### ISSUE THREAD

**Issue by rifatozkancomtr** [Crunchy-DL/Crunchy-Downloader#146]
![image](https://github.com/user-attachments/assets/c519a469-728a-47a3-aafa-990e709b753b)

Log mode enabled.
No proxy is being used.
Missing Refresh Token
Log mode enabled.
Missing Refresh Token
No proxy is being used.
Log mode enabled.
No proxy is being used.

**Comment by Elwador** [comment]
when does this happen? after a restart? if yes does it save a refresh token in the config folder in the token file?

**Comment by rifatozkancomtr** [comment]
> when does this happen? after a restart? if yes does it save a refresh token in the config folder in the token file?

Yes, it saves the Token but it constantly logs me out of my account. I deleted the token file and it was recreated, but the error persists.

**Comment by Elwador** [comment]
in the file is there a "refresh_token:"?

`it constantly logs me out of my account`
after a restart or while the programm is running?

**Comment by rifatozkancomtr** [comment]
I also reboot but not always.

![Screenshot_20241015_184224_Chrome](https://github.com/user-attachments/assets/6053961d-19c8-49c6-b811-462fb61881f3)

**Comment by Elwador** [comment]
in the cr_token.yml - is the refresh token (don't share it)
does it say "refresh_token:" or does it have a id to the right of it?

so it only happens sometimes?

**Comment by rifatozkancomtr** [comment]
> in the cr_token.yml - is the refresh token (don't share it) does it say "refresh_token:" or does it have a id to the right of it?
> 
> so it only happens sometimes?

Refresh Token appears empty. No, it always happens after a while.

**Comment by Elwador** [comment]
hm strange - is there a refresh token in the file right after you login in the downloader without doing anything else?

**Comment by rifatozkancomtr** [comment]
Now I'm logged back in. There is a Refresh Token. When I open the app the screen.

![Screenshot_20241015_192314_Chrome](https://github.com/user-attachments/assets/bd5bdd01-0535-4d88-babe-94d45b777b95)

I closed the app. 
I restarted the computer.
There doesn't seem to be any problem.
It will be logged out of the account within an hour.

I've been doing DNS forwarding on controld.com for a long time. 

This problem occurred after the last update.
